lib/power_action_utils.pm

```diff
@@ -81,12 +81,19 @@ sub reboot_x11 {
     my ($self) = @_;
     wait_still_screen;
     if (check_var('DESKTOP', 'gnome')) {
-        if (is_tumbleweed) {
+        # Get gnome version
+        x11_start_program('xterm -geometry 150x45+5+5', target_match => 'xterm');
+        become_root;
+        my $gnome_version = script_output("rpm -q --queryformat '%{VERSION}' gnome-shell");
+        type_string "killall xterm\n";
```

Comment:
`send_key `alt-f4';` should be enough

Reply:
SInce we're in the console already, killall doesn't hurt

Reply:
it's many more key presses and an unusual way to end a single xterm and not what we do in other cases as well. Tests should be written in a clean and simple way. I know that `killall xterm` is used in other cases where it might make sense, e.g. in tests/x11/sshxterm where one wants to kill all instances of xterm not knowing which window is focussed or where an "alt-f4" is ambiguous. I assume that @ggkioulis just took an existing module as inspiration, which is a good approach, but I am just saying: Other modules are a better source for inspiration here :)

Reply:
Well in this case, and this context: The system is about to be shut down :) So it doesn't really matter